libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/component/roadlabel/RoadNameViewBinderTest.kt

```diff
@@ -1,35 +0,0 @@
-package com.mapbox.navigation.dropin.component.roadlabel
```

Comment:
Will re-add binder tests once I figure out how to test `reloadOnChange`  result.